core/templates/base-components/base.module.ts

```diff
@@ -0,0 +1,148 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module having all essentials for creating a oppia page.
+ */
+
+// Modules.
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { CookieModule } from 'ngx-cookie';
+import { ToastrModule } from 'ngx-toastr';
+
+import { DirectivesModule } from 'directives/directives.module';
+import { SharedPipesModule } from 'filters/shared-pipes.module';
+import { I18nModule } from 'i18n/i18n.module';
+
+// Components.
+import { CreateActivityButtonComponent } from 'components/button-directives/create-activity-button.component';
+import { SocialButtonsComponent } from 'components/button-directives/social-buttons.component';
+import { AlertMessageComponent } from 'components/common-layout-directives/common-elements/alert-message.component';
+import { PromoBarComponent } from 'components/common-layout-directives/common-elements/promo-bar.component';
+import { SideNavigationBarComponent } from 'components/common-layout-directives/navigation-bars/side-navigation-bar.component';
+import { TopNavigationBarComponent } from 'components/common-layout-directives/navigation-bars/top-navigation-bar.component';
+import { LoadingMessageComponent } from './loading-message.component';
+import { OppiaFooterComponent } from './oppia-footer.component';
+import { WarningsAndAlertsComponent } from './warnings-and-alerts.component';
+
+// Directives.
+import { BaseContentComponent,
+  BaseContentNavBarBreadCrumbDirective,
+  BaseContentPageFooterDirective } from './base-content.component';
```

Comment:
Please either do 
```suggestion
import { BaseContentComponent, BaseContentNavBarBreadCrumbDirective, BaseContentPageFooterDirective } from './base-content.component';
```
or
```suggestion
import { 
  BaseContentComponent,
  BaseContentNavBarBreadCrumbDirective,
  BaseContentPageFooterDirective 
} from './base-content.component';
```

Reply:
done